includes/admin/class-wcmp-export-consignments.php

```diff
@@ -320,11 +321,19 @@ private function setRecipient(): void
             ->setEmail($this->recipient['email'])
             ->setPhone($this->recipient['phone']);
 
-        if ($postnl) {
+        $country = $this->consignment->getCountry();
+
+        if ($country === 'BE') {
+            if ($bpost) {
+                $this->consignment->setBoxNumber($this->recipient['number_suffix'] ?? null);
+                return;
+            }
+
             $this->consignment->setNumberSuffix($this->recipient['number_suffix'] ?? null);
-        } else {
-            $this->consignment->setBoxNumber($this->recipient['number_suffix'] ?? null);
+            return;
```

Comment:
Zodra je 1 if heb dan hoef je  `$this->consignment->setNumberSuffix .....` en de return niet te doen